tools/src/publish-packages/tasks/updateWorkspaceProjects.ts

```diff
@@ -22,6 +23,19 @@ export const updateWorkspaceProjects = new Task<TaskArgs>(
     logger.info('\n📤 Updating workspace projects...');
 
     const workspaceInfo = await Workspace.getInfoAsync();
+
+    // Append project templates as they can't be a yarn workspace.
```

Comment:
```suggestion
    // Append project templates as they are not covered yarn workspace.
```

nit: we just not include templates in workspace. that said, "can't be" looks not quite correct!?